.github/workflows/checkAndSubmitAddonMetadata.yml

```diff
@@ -179,8 +179,175 @@ jobs:
       uses: peter-evans/close-issue@v3
       with:
         issue-number: ${{ inputs.issueNumber }}
+
+  createReviewComment:
+    continue-on-error: true
+    # jq for windows has issues parsing multiline strings (e.g. CRLF),
+    # use linux instead.
+    runs-on: ubuntu-latest
+    needs: [checkMetadata, getAddonFilename]
+    strategy:
+      matrix:
+        python-version: [ 3.11 ]
+    permissions:
+      contents: write
+      discussions: write
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.issueAuthorName }}${{ inputs.issueNumber }}
+    - name: Get add-on name and version
+      id: getAddonNameAndVersion
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const fs = require('fs')
+          const addonFilename = ${{ needs.getAddonFilename.outputs.addonFileName }}
+          const contents = fs.readFileSync(addonFilename)
+          const metadata = JSON.parse(contents)
+          const addonId = metadata.addonId
+          core.setOutput('addonId', addonId)
+          const addonName = metadata.displayName
+          core.setOutput('addonName', addonName)
+          const addonVersion = metadata.addonVersionName
+          core.setOutput('addonVersion', addonVersion)
```

Comment:
Sorry, I know this suggestion is late.

Could you move all the longer scripts to files (e.g. 10 lines of code or longer)?

This is kind of the norm with [getData](https://github.com/nvaccess/addon-datastore/blob/master/.github/workflows/getData.js)